### DISCUSSION TITLE
i need help to create Custom Notifications popup in POS

### DISCUSSION THREAD

**Discussion by confizwaqar** [microsoft/Dynamics365Commerce.Solutions#329] [Q&A]
I need help creating a Custom notification popup in POS. The popup will be populated on the Cart screen and based on the custom values from the CommerceRuntime CRT(RTS or Custom Value) in store commerce d365. A popup will appear after every 10 mins or on getting some new values.

**Comment by dsdharmen91** [comment]
Hi you can create by customization. you can reach to dsdharmen91@gmail.com
